test/issues/0468.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+
+var assert = require('assert');
+var yaml = require('../..');
+
+test('should not indent arrays an extra level when disabled', function () {
+  /* eslint-disable max-len */
+  var output = yaml.dump([ { a:'a_val', b:'b_val' }, { a:'a2_val', expanded:true, items: [ { a:'a_a_val', b:'a_b_val' } ] } ], { noArrayIndent: true });
```

Comment:
Please, format this to multiline form first, as been requested (i provided sample). Lint rules can be disabled only when following those is not acceptable. Not this case.